spec/system/admin/reports/revenues_by_hub_spec.rb

```diff
@@ -10,22 +10,67 @@
       :completed_order_with_totals,
       completed_at: 2.days.ago,
       order_cycle:,
-      distributor: create(:enterprise, name: "Hub 1",
-                                       owner: create(:user, email: "email@email.com")),
+      distributor: distributor1
     )
   end
+  let(:order_with_voucher_tax_included) do
+    create(
+      :order_with_taxes,
+      completed_at: 2.days.ago,
+      order_cycle:,
+      distributor: distributor2,
+      product_price: 110.0,
+      tax_rate_amount: 0.1,
+      included_in_price: true,
+      tax_rate_name: "Tax 1"
+    )
+  end
+  let(:order_with_voucher_tax_excluded) do
+    create(
+      :order_with_taxes,
+      completed_at: 2.days.ago,
+      order_cycle:,
+      distributor: distributor3,
+      product_price: 110.0,
+      tax_rate_amount: 0.1,
+      included_in_price: true,
```

Comment:
It should be `included_in_price: false` to get tax excluded from price. In this case it's not affecting the result as you are mocking `VoucherAdjustmentsService`